example.js

```diff
@@ -26,6 +27,9 @@ import {
 // load the OTP configuration
 import otpConfig from './config.yml'
 
+const MyLegIcon = TriMetLegIcon // For testing, you can use () => <Ferry />
+const MyModeIcon = () => TriMetModeIcon // For testing, you can use () => <AerialTram />
```

Comment:
Can you add commented out lines below this that people can easily uncomment to try out the testing overrides? Also, it would be good to describe what the testing will show (and where).